### DISCUSSION TITLE
error when trying to install on raspberry pi OS 64 bit

### DISCUSSION THREAD

**Discussion by dylanb-101** [coder/code-server#4917] [Q&A]
when trying to install via the install script it installs fine but when trying to run code-server I get this error: 
`Error: Cannot find module '@node-rs/argon2-linux-arm64-gnu'` 
I tried installing the package and then running again but got the same error. Does anyone know how to fix it?

**Comment by lg-nineoneninethreenine** [comment]
Can you try checking if you are installing the latest version?

**Comment by silviaclaire** [comment]
Should be fixed with next release? #4804

**Reply by lg-nineoneninethreenine** [reply]
Yup.
